utils/oscap.8

```diff
@@ -191,6 +191,11 @@ Do not validate input/output files.
 Do not validate digital signatures in digitally signed SCAP source data streams.
 .RE
 .TP
+\fB\-\-enforce-signature\fR
+.RS
+Process only digitally signed SCAP source data streams.
```

Comment:
Maybe add for clarity that data streams with no signature will be rejected.

Reply:
Done